development/mimir-monolithic-mode/config/otel-collector-otlp-push-config.yaml

```diff
@@ -0,0 +1,37 @@
+# Based on information on https://opentelemetry.io/docs/collector/configuration/
+receivers:
+  # Data sources: traces, metrics, logs
+  otlp: # Allow sending data via OTLP protocol
+    protocols:
+      grpc:
+      http:
+
+  # Data sources: metrics
+  # NOTE: currently (0.84.0) does not support transforming prometheus metrics to OTLP metrics
```

Comment:
I couldn't get it to work and I'm basing this on google answers, can someone confirm ? @jesusvazquez

Reply:
Thanks to @jpkrohling we got it working. Wrong config file used.